client-src/default/index.js

```diff
@@ -221,7 +221,8 @@ if (
 ) {
   protocol = self.location.protocol;
 }
-
+// for server side rendering
+port = SOCKJS_PORT || port; // eslint-disable-line
```

Comment:
We have https://webpack.js.org/configuration/dev-server/#devserversockpath i think we need implement `socksjsPort` option here

Reply:
wait a minutes i will provide a minimum reproducible test repo
your idea is add a socksjsPort options?

Reply:
https://github.com/zhangyuang/egg-ssr
@evilebottnawi 
this is the minimum reproducible test repo
and i use this changes to a single webpack-dev-server named yumi-webpack-dev-server temporary

Reply:
Yes, we already have path, but you try to use port in `sockPath`?

Reply:
i don't use sockpath now, i don't know whether i can use port in sockPath
i find socket port use location.port now, but i hope use webpack-dev-server listen port
because in ssr,the tab i opening is start by my own node server not webpack-dev-server